pygmt/src/which.py

```diff
@@ -23,11 +23,13 @@ def which(fname, **kwargs):
     defined), (3) in $GMT_DATADIR (if defined), or (4) in $GMT_CACHEDIR
     (if defined).
 
-    ``fname`` can also be a downloadable file (either a full URL, a
-    `@file` special file for downloading from the GMT Site Cache, or
-    `@earth_relief_*` topography grids). In these cases, use parameter
-    ``download`` to set the desired behavior. If ``download`` is not used
-    (or False), the file will not be found.
+    ``fname`` can also be a downloadable file (either a complete URL, a
+    @file for downloading from the GMT data server, or @earth_relief_xxy
```

Comment:
We don't have to mention the `@earth_relief_xxy` file here because it's just one of the "remote datasets".

Reply:
Please see commit https://github.com/GenericMappingTools/pygmt/pull/2519/commits/7cb975c7e7002cd3f84ae142883a3d1ce6255cbd for this change.

Reply:
Btw @seisman shoud I also update the upstream GMT documentation at https://docs.generic-mapping-tools.org/dev/gmtwhich.html#g?

Reply:
> Btw @seisman shoud I also update the upstream GMT documentation at https://docs.generic-mapping-tools.org/dev/gmtwhich.html#g?

Sounds good.

Reply:
I just saw that my initial suggested formulation in https://github.com/GenericMappingTools/gmt/pull/7426/commits/8b38a5dc4758b2e7e201a6b12100e34a53481cff did not explicitly mention the earth relief dataset. But based on the review in https://github.com/GenericMappingTools/gmt/pull/7426#pullrequestreview-1404146936 I changed the formulation to still explicitly mention the earth relief dataset.

Reply:
I see. I think either is fine.

Reply:
Then we will probably leave the upstream GMT documentation as is.